lib/node.js

```diff
@@ -239,7 +239,7 @@ declare class child_process$ChildProcess extends events$EventEmitter {
   disconnect(): void;
   kill(signal?: string): void;
   send(
-    message: Object,
+    message: any,
```

Comment:
Could you change this to `mixed`? It won't really make a difference since this is an input position, but I'd prefer to avoid `any` where possible.